binder/start

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -x
+export DISPLAY=:99.0
+export PYVISTA_OFF_SCREEN=true
+Xvfb :99 -screen 0 1024x768x24 > /dev/null 2>&1 &
+sleep 3
+set +x
+exec "$@"
```

Comment:
none of the above is needed

Reply:
This script came from the pyvista docs, and I was getting the error mentioned above without this script.

Reply:
Yes that is when you use regular vtk not the osmesa one.

Reply:
I'm including `vtk-osmesa` in `requirements.txt`, but I'm not able to get it working without xvfb and the start script. Could you show me what you mean?

Reply:
if xvfb is needed, that is because vtk-osmesa is not the one that is running. It is the default vtk that need X.
Also vtk and vtk-osmesa technically are the same... So if you don't remove vtk before installing vtk-osmesa, who knows what you are getting since they write the same files/directories...

Reply:
If this is the case, then it should be the case that we can just `pip install vtk` rather than `vtk-osmesa`, and it should still work, right?

Reply:
well if it working now then you are all-set since osmesa is not used.
But the point of osmesa was to skip xvfb and other system stuff

Reply:
BTW, `vtk` is a dependency of `pyvista` so we don't even do `pip install vtk`